api/v1/storageoscluster_types.go

```diff
@@ -230,6 +231,11 @@ type StorageOSClusterKVBackend struct {
 	Backend string `json:"backend,omitempty"`
 }
 
+// PVCMetrics contains the configuration for the PVC metrics.
+type PVCMetrics struct {
```

Comment:
nit: I would just name this `Metrics`. It's likely we'll have lots of different metrics here, not just metrics for PVCs 

Please ignore me if @Ricardo-Osorio has suggested something better \:)

Reply:
No, that's a good point 👍

Reply:
Discussed the naming with @Ricardo-Osorio beforehand, so will ignore :)